server/src/main/java/dev/slimevr/tracking/processor/BoneInfo.java

```diff
@@ -13,7 +16,12 @@ public class BoneInfo {
 	public final TransformNode headNode;
 	public final TransformNode tailNode;
 	public float length;
-	private static final Quaternion FOOT_OFFSET = Quaternion.X_90_DEG;
+	private static final Quaternion FOOT_OFFSET = new EulerAngles(
+		EulerOrder.YZX,
+		FastMath.HALF_PI,
+		0f,
+		0f
+	).toQuaternion();
```

Comment:
I'm sure this can be done without euler angles?

Reply:
If you think that this improves readability, then I'm fine with it. I just don't think needing euler angles for something so simple is necessarily beneficial otherwise.

Reply:
its converted to quaternion, there is nothing complex on this?

Reply:
It's not complex, just if it can be simpler then I'd take it as such. This is a non-issue.